crypto/src/proofs/transparent.rs

```diff
@@ -397,21 +400,19 @@ pub struct SwapProof {
     pub g_d: decaf377::Element,
     // The transmission key for the address.
     pub pk_d: ka::Public,
-    // The value of the note.
-    pub value: Value,
+    // The value of the swap NFT.
+    pub swap_nft_value: Value,
```

Comment:
Since the amount will always be 1, rather than recording an amount/asset id pair (`Value`), we could just record
```
pub swap_nft_id: asset::Id,
```